### PR TITLE
feat: added widget test for Login Page

### PR DESCRIPTION
**Pull request by arteevraina**
**What kind of change does this PR introduce?**

This PR adds widget test for `LoginPage` .

**Did you add tests for your changes?**

Yes, I have added the widget test.

**Does this PR introduce a breaking change?**

No

**Other information**
ref: #326

### CODE REVIEW

File: test/widget_tests/login_page_test.dart [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+      expect(
+        find.text("Set URL"),
+        findsNothing,
+      );
+      expect(
+        find.text("URL SAVED!"),
+        findsOneWidget,
+      );
+
+      // Tap on the createAccountButton.
```
Comment (Sagar2366) [inline test/widget_tests/login_page_test.dart:231]: Please update comment to tap on loginButton

**Comment by Sagar2366** [comment]
@arteevraina so far this is good addition for testing home page screen.
More test cases  need to be added to make sure this screen is functioning properly.
This is already big PR to review. We will break down next widget testing into small issues and PR's.

**Comment by arteevraina** [comment]
Sure thing @Sagar2366! Can you please elaborate on some cases which we need to test further in `login_page.dart`.

**Comment by Sagar2366** [comment]
> Sure thing @Sagar2366! Can you please elaborate on some cases which we need to test further in `login_page.dart`.

1. Protocol selection button is available or not and it is changing dropdown values
2. On changing protocol "URL Saved!" text changes to "set URL"

**Comment by arteevraina** [comment]
Cool ! Should I test this in my next PR ? This PR is ready as per the changes requested by you.

**Review by Sagar2366 — APPROVED** [review]

**Pull request merged**